oauth/client_test.go

```diff
@@ -0,0 +1,234 @@
+package oauth
+
+import (
+	"bytes"
+	assert "github.com/stretchr/testify/require"
+	stripe "github.com/stripe/stripe-go"
+	_ "github.com/stripe/stripe-go/testing"
+	"io/ioutil"
+	"net/http"
+	"testing"
```

Comment:
This may be a local stylistic convention, but would you mind breaking the imports into two groups? That's generally how we've done it here.

1. Al the "built-ins" like `bytes`, `io/ioutil`, etc.
2. All the GOPATH packages like `assert "github.com/stretchr/testify/require"`, `stripe "github.com/stripe/stripe-go"`, `_ "github.com/stripe/stripe-go/testing"`.

(You can find examples in most other files in the project.)

Gofmt is smart enough to support this fine.

Reply:
Got it. Thanks for the heads up :)

Reply:
I think I got all of these cleaned up.